Source/WebCore/dom/DocumentMarkerController.cpp

```diff
@@ -461,6 +466,22 @@ Vector<RenderedDocumentMarker*> DocumentMarkerController::markersFor(Node& node,
     return result;
 }
 
+void DocumentMarkerController::forEach(OptionSet<DocumentMarker::MarkerType> types, const Function<bool(Node&, RenderedDocumentMarker&)> function)
```

Comment:
I think this should be more like forEachofType or something else that indicates that it's not jut for all the markers. Or maybe I'm thinking too Obj-C style names here.

Reply:
Sounds good – will rename to `forEachOfTypes` (since multiple types could be passed in the set).